packages/xo-server/src/models/user.mjs

```diff
@@ -34,4 +34,14 @@ export class Users extends Collection {
     // Adds the user to the collection.
     return /* await */ this.add(properties)
   }
+
+  async updateIfNotExists(properties) {
+    const { email } = properties
+
+    if (await this.exists({ email })) {
```

Comment:
I think we shouldn't test this if email is undefined ( not updating this property )
```suggestion
    if (email !=undefined && await this.exists({ email })) {
```

Reply:
not needed since updateUser start by populating